### PR TITLE
fix: db helpers to be used with timestamps

### PR DESCRIPTION
**Pull request by dni**
those helpers are used in boltz extension and they did not take dates into consideration yet

### CODE REVIEW

File: lnbits/db.py [hunk 1]
```diff
@@ -65,6 +65,13 @@ def compat_timestamp_placeholder():
         return "?"
 
 
+def get_placeholder(type_: type) -> str:
+    if type_ == datetime.datetime:
+        return compat_timestamp_placeholder()
+    else:
```
Comment (motorina0) [inline lnbits/db.py:71]: no need for `else`

**Review by motorina0 — APPROVED** [review]

**Comment by motorina0** [comment]
LGTM, but unit tests fail

File: lnbits/db.py (hunk 1 above)
Comment (motorina0) [reply lnbits/db.py:71]: maybe this?

```py
def get_placeholder(model: Any, field: str) -> str:
   type_ = model.__fields__[field].type_
   ...
```

then call like this:
```py
placeholder = get_placeholder(self.model, self.field)
```

looks cleaner and encapsulates the object reflection logic in one place

**Review by prusnak — APPROVED** [review]

**Comment by prusnak** [comment]
added one commit with `get_placeholder` refactor as suggested by @motorina0

**Pull request merged**